bundles/mapping/dimension-change/resources/locale/fi.js

```diff
@@ -0,0 +1,12 @@
+Oskari.registerLocalization(
+    {
+        "lang": "fi",
+        "key": "dimensionchange",
+        "value": {
+            "to3Dview": "Siirry 3D-näkymään. Käytettävissä vain osa karttatasoista ja työkaluista.",
+            "backTo2Dview": "Palaa Paikkatietoikkunan oletusnäkymään.",
```

Comment:
Paikkatietoikkuna reference here that should be removed if we merge this to Oskari. Maybe we should consider moving the dimension change to Paikkatietoikkuna repository instead of having it on oskari-frontend or make a more generic "appsetup selector" bundle for Oskari.

Reply:
Let's go with having the dimension-change bundle in oskari-frontend for now and just remove the PTI reference from the toolip.